orc8r/cloud/go/services/state/indexer/reindex/reindexer_singleton_test.go

```diff
@@ -0,0 +1,169 @@
+/*
+Copyright 2020 The Magma Authors.
+
+This source code is licensed under the BSD-style license found in the
+LICENSE file in the root directory of this source tree.
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package reindex_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/stretchr/testify/mock"
+	"github.com/stretchr/testify/require"
+
+	"magma/orc8r/cloud/go/clock"
+	configurator_test_init "magma/orc8r/cloud/go/services/configurator/test_init"
+	configurator_test "magma/orc8r/cloud/go/services/configurator/test_utils"
+	device_test_init "magma/orc8r/cloud/go/services/device/test_init"
+	directoryd_types "magma/orc8r/cloud/go/services/directoryd/types"
+	"magma/orc8r/cloud/go/services/orchestrator/obsidian/models"
+	"magma/orc8r/cloud/go/services/state/indexer"
+	"magma/orc8r/cloud/go/services/state/indexer/reindex"
+	state_test_init "magma/orc8r/cloud/go/services/state/test_init"
+	state_test "magma/orc8r/cloud/go/services/state/test_utils"
+)
+
+func TestSingletonRun(t *testing.T) {
+	// Make nullimpotent calls to handle code coverage indeterminacy
+	reindex.TestHookReindexSuccess()
```

Comment:
Let's do this then: let's leave the success hook in place for this PR, and refactor it out at the end of this milestone by using `sendUpdate` instead of the test hook

Reply:
Added a comment!